src/packages/ufm/components/ufm-render/ufm-render.element.ts

```diff
@@ -59,6 +59,14 @@ export class UmbUfmRenderElement extends UmbLitElement {
 			pre {
 				overflow: auto;
 			}
+
+			:host > :first-child {
+				margin-block-start: 0;
```

Comment:
I'm happy with reducing the excessive margins (caused by the `<p>` tags). 👍 

I'm thinking that a small top margin (3px) would be a good fit. What do you think @skttl?

```suggestion
				margin-block-start: var(--uui-size-1);
```

Reply:
Hi @leekelleher just chipping in, I think it would be good to have a minimalistic approach on the space taken by UFM-render. I think its better if the specific implementation append the margin to it, if its needed there — in that way enabling UFM to be used in very narrow spaces, or in cases where other things makes up the white-space. :-)